util/utility.js

```diff
@@ -131,7 +131,7 @@ function buildMap() {
     */
 }
 
-const isRadiant = (p) => p.player_slot < 128;
+const isRadiant = (playerSlot) => playerSlot < 128;
```

Comment:
yea, it wasn't working right like that and I think that in general functions should only pass around the data they care about. In this case, we literally just care about checking the player slot, so there is no reason to pass around the entire match object. I think we can just be aware that all the real data from the server is snake_case, maybe we can put something in the readme.